src/main/java/fi/dy/masa/litematica/schematic/LitematicaSchematic.java

```diff
@@ -65,32 +50,25 @@
 import fi.dy.masa.litematica.selection.Box;
 import fi.dy.masa.litematica.util.BlockUtils;
 import fi.dy.masa.litematica.util.EntityUtils;
-import fi.dy.masa.litematica.util.FileType;
-import fi.dy.masa.litematica.util.NbtUtils;
 import fi.dy.masa.litematica.util.PositionUtils;
-import fi.dy.masa.litematica.util.ReplaceBehavior;
-import fi.dy.masa.litematica.util.SchematicPlacingUtils;
 import fi.dy.masa.litematica.util.WorldUtils;
-import fi.dy.masa.malilib.gui.Message.MessageType;
-import fi.dy.masa.malilib.interfaces.IStringConsumer;
-import fi.dy.masa.malilib.util.Constants;
-import fi.dy.masa.malilib.util.FileUtils;
-import fi.dy.masa.malilib.util.InfoUtils;
-import fi.dy.masa.malilib.util.IntBoundingBox;
-import fi.dy.masa.malilib.util.NBTUtils;
-import fi.dy.masa.malilib.util.StringUtils;
+import fi.dy.masa.litematica.util.*;
 
 public class LitematicaSchematic
 {
     public static final String FILE_EXTENSION = ".litematic";
     public static final int SCHEMATIC_VERSION_1_13_2 = 5;
+    public static final int SCHEMATIC_VERSION_1_20_5 = 7;
+    public static final int MINECRAFT_DATA_VERSION_1_12   = 1139; // MC 1.12
     public static final int MINECRAFT_DATA_VERSION_1_13_2 = 1631; // MC 1.13.2
-
+    public static final int MINECRAFT_DATA_VERSION_1_20_5 = 3834; // MC 1.20.5-rc.1
     public static final int MINECRAFT_DATA_VERSION = SharedConstants.getGameVersion().getSaveVersion().getId();
-    public static final int SCHEMATIC_VERSION = 6;
+    public static final int SCHEMATIC_VERSION = 7;
```

Comment:
Increments SCHEMATIC_VERSION due to Data Components being in the .litematic files.